### DISCUSSION TITLE
Incorrect spacegroup on Si diamond structure

### DISCUSSION THREAD

**Discussion by rudytak** [spglib/spglib#498] [Q&A]
Hi everyone,
I am having some issues getting the correct symmetry operations/spacegroup outputs from some of the materials I'm working in. My goal is to recreate the auto-generated wien2k symmetry operations, but I wanted to start on something small.

I chose the standard example of a cubic Si crystal like this one here: https://next-gen.materialsproject.org/materials/mp-149

I then tried getting the spacegroup (which should be Fd-3m aka 227), but I'm getting R-3m (166) instead. Would anyone be able to help out please?

My code snippet is:

```py
import spglib
lattice = [[1.0, 0.0, 0.0],
            [0.0, 1.0, 0.0],
            [0.0, 0.0, 1.0]]
positions = [[0.0, 0.0, 0.0], # Si
            [0.25, 0.25, 0.25]] # Si
numbers = [1, 1]
cell = (lattice, positions, numbers)

# expecting 277 here
print(spglib.get_spacegroup(cell))
```

**Comment by atztogo** [comment]
You have to provide all atoms in your choice of the unit (primitive) cell.
